bundles/org.eclipse.passage.loc.billing.core/src/org/eclipse/passage/loc/billing/internal/core/BillingServiceImpl.java

```diff
@@ -0,0 +1,32 @@
+package org.eclipse.passage.loc.billing.internal.core;
+
+import java.util.Date;
+import java.util.HashSet;
+import java.util.Set;
+
+import org.eclipse.passage.lic.users.UserLicenseDescriptor;
+
+public class BillingServiceImpl implements BillingService {
+
+	@Override
+	public Set<String> getUserLicenses(Iterable<? extends UserLicenseDescriptor> licenses) {
```

Comment:
So, this one will return which plans were used by the given issued licenses. Does not look like billing: no idea how to understand the fee.